test/acceptance/editors-integration-test.js

```diff
@@ -11,7 +11,11 @@ function setForBasicDebuggerError(project) {
 function run(project, args, options = {}) {
   options.reject = false;
   options.cwd = options.cwd || project.path('.');
-  return execa.sync(require.resolve('../../bin/ember-template-lint.js'), args, options);
+
+  // for windows we need to call node <script.js>
+  // however the node method call hangs sometimes on linux, but works smoothly on windows
+  let runFunc = process.platform === 'win32' ? execa.node : execa.sync;
+  return runFunc(require.resolve('../../bin/ember-template-lint.js'), args, options);
```

Comment:
The same comment than https://github.com/ember-template-lint/ember-template-lint/pull/1284/files#r410979081 would apply here.